core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/CameraMarkerTracker.java

```diff
@@ -119,74 +119,74 @@ private boolean transformsAreWithinReasonableEpsilonOfEachOther(AffineMatrix4x4
   }
 
   private void animateToTargetView() {
-    AffineMatrix4x4 currentTransform = this.perspectiveCamera.getAbsoluteTransformation();
-    AffineMatrix4x4 targetTransform = this.activeMarker.getTransformation(org.lgna.story.implementation.AsSeenBy.SCENE);
+    AffineMatrix4x4 currentTransform = perspectiveCamera.getAbsoluteTransformation();
+    AffineMatrix4x4 targetTransform = activeMarker.getTransformation(org.lgna.story.implementation.AsSeenBy.SCENE);
 
-    if (this.pointOfViewAnimation != null) {
-      this.doEpilogue = false;
-      this.pointOfViewAnimation.complete(null);
-      this.doEpilogue = true;
+    if (pointOfViewAnimation != null) {
+      doEpilogue = false;
+      pointOfViewAnimation.complete(null);
+      doEpilogue = true;
     }
     if (transformsAreWithinReasonableEpsilonOfEachOther(currentTransform, targetTransform)) {
-      startTrackingCamera(CameraMarkerTracker.this.perspectiveCamera, CameraMarkerTracker.this.activeMarker);
+      startTrackingCamera(perspectiveCamera, activeMarker);
     } else {
-      Transformable cameraParent = (Transformable) this.perspectiveCamera.getParent();
-      this.pointOfViewAnimation = new PointOfViewAnimation(cameraParent, AsSeenBy.SCENE, currentTransform, targetTransform) {
+      Transformable cameraParent = (Transformable) perspectiveCamera.getParent();
+      pointOfViewAnimation = new PointOfViewAnimation(cameraParent, AsSeenBy.SCENE, currentTransform, targetTransform) {
         @Override
         protected void epilogue() {
```

Comment:
This was the one set of this's that i was unsure about changing...

Reply:
I like it THIS way.